### ISSUE TITLE
tmux visual section and yank bindings

### ISSUE THREAD

**Issue by khornberg** [JarrodCTaylor/dotfiles#42]
I have tmux `2.3_3` via Homebrew installed. The current key bindings for visual section and yank do not work for me on OS 10.12.3.

Replacing lines 33-35 in `mac-tmux/tmux.conf`
```
bind-key -Tcopy-mode-vi 'v' send -X begin-selection
bind-key -Tcopy-mode-vi 'y' send -X copy-pipe "reattach-to-user-namespace pbcopy"
bind-key -Tcopy-mode-vi 'Escape' send -X cancel
```
makes them work again.

References:
https://github.com/tmux/tmux/issues/592
https://github.com/tmux/tmux/commit/76d6d3641f271be1756e41494960d96714e7ee58

**Comment by mattjmorrison** [comment]
Not a problem on OS X 10.9.5, tmux 1.9a, & iTerm 1.0.0

**Comment by JarrodCTaylor** [comment]
Closed by PR #45
